csrankings-m.csv

```diff
@@ -2317,7 +2317,7 @@ Monika Henzinger,University of Vienna,https://cs.univie.ac.at/monika.henzinger,N
 Monika Rauch,University of Vienna,https://cs.univie.ac.at/monika.henzinger,NXbggxYAAAAJ
 Monika Rauch Henzinger,University of Vienna,https://cs.univie.ac.at/monika.henzinger,NXbggxYAAAAJ
 Monique Ross,Florida International University,https://www.cis.fiu.edu/faculty-staff/ross-monique,LtfU7CgAAAAJ
-Monowar Hasan,Wichita State University,https://www.wichita.edu/profiles/academics/engineering/SoC/Hasan-Monowar.php,lLMWtnkAAAAJ
+Monowar Hasan,Washington State University,https://school.eecs.wsu.edu/faculty/profile/?nid=monowar.hasan,lLMWtnkAAAAJ

```

Comment:
FYI, the site points to a web page which produces a 404 (https://monowarhasan.info/).